package.json

```diff
@@ -76,9 +76,7 @@
     "cross-env": "^5.1.4",
     "jest-dom": "^3.0.0"
   },
-  "availityWorkflow": {
-    "plugin": "@availity/workflow-plugin-react"
-  },
+  "availityWorkflow": true,
```

Comment:
I think we should also update the `@availity/workflow` dependency and remove `@availity/workflow-plugin-react`. And we should eventually get all the regular dependencies updated, but this is good for now